src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div id="name-request-summary">
+    <v-row >
+      <v-col id="name-request-info" cols="4">
+        <ul>
+          <li> <label>
+          <strong>Name Request</strong>
+        </label></li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li> Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>
+            <v-icon v-if="nameRequestDetails.status === NameRequestStates.APPROVED" color="green"
+            class="nr-status-icon">mdi-check</v-icon>
+            Status: {{ formatStatusText(nameRequestDetails.status) }}
+          </li>
+          <li id="condition-consent">
+            <v-icon v-if="conditionConsent() === 'Received' || conditionConsent() === 'Not Required'"
+            color="green" class="nr-status-icon">mdi-check</v-icon>
+            <v-icon v-if="conditionConsent() === 'Not Received'"
+            color="red" class="nr-status-icon">mdi-close</v-icon>
+            Condition/Consent: {{ conditionConsent() }}
+          </li>
+        </ul>
+      </v-col>
+      <v-col id="name-request-applicant-info" cols="8">
+        <ul>
+          <li><label>
+          <strong>Name Request Applicant</strong>
+        </label></li>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ nameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ nameRequestApplicant.phoneNumber | VMask('(###) ###-####')}}</li>
```

Comment:
Are we guaranteed that phone numbers are 10 digits?
This is cool but may not work in all situations.
What if phoneNumber is null?
Also this doesn't handle extension (per mockup). I don't even know what that looks like in the NR.

Reply:
Also, why not use the Vuetify text field mask here instead of a new package/filter?

Reply:
This is not a text field. I am not sure whether the  phone number is allways 10 digits. As per the wireframe and as per the phone number captured in Business Contact Info it is 10 digits. However, I will c onfirm with Scott.
I checked the NR data and Idint find an extension . I dint find the support for extension in the Step 1 NR display as well. I will confirm with Scott

Reply:
Looks like text to me in the NR response:
`"phoneNumber":"2507876565",`

Reply:
Checked with Scott. Extension is not required. There is an open question of international phone number in future. He liked the way it looks now and when we have an answer for the international phone number this can be revisited